test/client_test.cc

```diff
@@ -93,6 +93,14 @@ class ClientTest : public Envoy::BaseIntegrationTest,
     return fmt::format("http://{}:{}/", address, port_);
   }
 
+  const char* getAddressFamilyOptionString() {
+    auto ip_version = GetParam();
+    RELEASE_ASSERT(ip_version == Envoy::Network::Address::IpVersion::v4 ||
```

Comment:
Don't we want to `throw` and print the error in the main CLI driver in a user friendly way?

Reply:
We do so, see `ParseAddressFamilyBadValues` below. This is just for test.

Reply:
(this assert is primarily meant to catch when it is time for maintenance, `GetParam()` would be returning something we didn't expect.